### PR TITLE
IA-1689: drag and drop

### PR DESCRIPTION
**Pull request by beygorghor**
PR using last bluesquare component sortable table and list.
You can test drag and drop on a workflow detail page an try to change follow ups order (Save is not implemented yet)

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests

## Changes

- adapt follow ups columns to fit SortableTable props
- remove table component
- adding list exemple

## How to test

Open workflow version detail page, make sure you have multiple followups for this version.

Try to change orders of follow ups

To test the list feature you need to go to `hat/assets/js/apps/Iaso/domains/workflows/details.tsx`and uncomment lines 123 to 134

## Print screen / video

https://user-images.githubusercontent.com/12494624/207059688-4197a195-57aa-4381-9d54-2d0193a96fdc.mov

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/workflows/details.tsx [hunk 1]
```diff
@@ -98,39 +120,28 @@ export const Details: FunctionComponent<Props> = ({ router }) => {
...
+                        className={classes.fullWidth}
+                        title={formatMessage(MESSAGES.followUps)}
+                    >
+                        <SortableList
+                            items={followUps}
+                            onChange={handleSortChange}
+                            RenderItem={props => <SortableItem {...props} />}
+                        />
+                    </WidgetPaper>
+                </Box> */}
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/workflows/details.tsx:134]: ?

File: hat/assets/js/apps/Iaso/domains/workflows/messages.ts [hunk 2]
```diff
@@ -91,7 +91,7 @@ const MESSAGES = defineMessages({
     },
     mapping: {
         defaultMessage: 'Mapping',
-        id: 'i,aso.workflows.mapping',
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/workflows/messages.ts:94]: lol

**Review by quang-le — COMMENTED** [review]
Columns for "changes" are still not sorted in the backend (though the url updates correctly). Is it already part of a later backend PR?

File: hat/assets/js/apps/Iaso/domains/workflows/details.tsx (hunk 1 above)
Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/workflows/details.tsx:134]: This is to test the other component if yo want, ill remove it after, see how to test !

**Comment by beygorghor** [comment]
> Columns for "changes" are still not sorted in the backend (though the url updates correctly). Is it already part of a later backend PR?

Yes this has still to be done normally

**Review by quang-le — APPROVED** [review]

**Review by codeKameleon — APPROVED** [review]

**Pull request merged**